background_r: 0
background_g: 0
background_b: 0